recipes/fastcov/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
```

Comment:
Please use the name and version variable thingies, e.g. for centrifuge

```
{% set name = "centrifuge" %}
{% set version = "1.0.4.1" %}
```

And then use that throughout with 

```
name: {{ name|lower }}
version: {{ version }}
```

Point 3 here: https://bioconda.github.io/tutorials/2024-adding-bioinformatic-software-to-bioconda.html#writing-the-recipe

This will allow autobumping